tests/test_histogram_parser.py

```diff
@@ -84,6 +85,15 @@ def histogram_parser(version, usecounter_optout):
         for field in REQUIRED_DETAILS:
             assert field in data['details']
 
+        # Default multistore if unspecified is just "main"
+        if name == "TELEMETRY_TEST_EXPONENTIAL":
+            assert ["main"] == data['details']['record_into_store']
+
+        # If multiple stores set, they should be both listed
+        if name == "HISTOGRAM_WITH_MULTISTORE":
+            assert "main" in data['details']['record_into_store']
+            assert "store2" in data['details']['record_into_store']
```

Comment:
Now I'm going to get persnickety and ask that you assert that it's _only_ `main` and `store2` that are in the list. : )